koji-setup/mash.sh

```diff
@@ -94,18 +95,18 @@ if [[ -e "$MASH_TRACKER_FILE" ]]; then
 else
 	MASH_BUILD_NUM=0
 fi
-KOJI_TAG="${KOJI_TAG:-"dist-clear"}"
 KOJI_REPO_PATH="$(realpath "$KOJI_DIR/repos/$KOJI_TAG-build/latest")"
 KOJI_BUILD_NUM="$(basename "$KOJI_REPO_PATH")"
 if [[ "$MASH_BUILD_NUM" -ne "$KOJI_BUILD_NUM" ]]; then
 	rm -rf "$MASH_DIR_NEW"
 	mkdir -p "$MASH_DIR_NEW"
 	create_dist_repos "$MASH_DIR_NEW"
-	if [[ -e "$MASH_TRACKER_DIR" ]]; then
-		mv "$MASH_TRACKER_DIR" "$MASH_DIR_OLD"
+	if [[ -e "$MASH_TRACKER_DIR/$KOJI_TAG" ]]; then
+		mv "$MASH_TRACKER_DIR/$KOJI_TAG" "$MASH_DIR_OLD/$KOJI_TAG"
 	fi
-	mv "$MASH_DIR_NEW" "$MASH_TRACKER_DIR"
+	mv "$MASH_DIR_NEW/$KOJI_TAG" "$MASH_TRACKER_DIR"
```

Comment:
Similarly with `$MASH_TRACKER_DIR` being used as a subdir, the script will need to ensure it is created first.